src/ctl/ctl.h

```diff
@@ -0,0 +1,207 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright 2016-2020, Intel Corporation */
+
+/*
+ * ctl.h -- internal declaration of statistics and control related structures
+ */
+
+#ifndef PMDK_CTL_H
```

Comment:
pls update all define's to use naming like: `UMF_*_H`

Reply:
Done

Reply:
not done, e.g. `CTL_DEBUG_H`

Reply:
Look once again, this snippet is outdated

Reply:
> CTL_DEBUG_H

this is not about the snippet - I'm saying it's not done for all defines - e.g. pls grep for `CTL_DEBUG_H`

Reply:
Done.